src/client.js

```diff
@@ -270,25 +271,43 @@ Client.prototype.input = function(data) {
 	var client = this;
 	var text = data.text.trim();
 	var target = client.find(data.target);
+
 	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+		text = "say " + text;
+	} else {
+		text = text.substr(1);
+
+		// If after removing first / we still have a /, send it as a message
+		if (text.charAt(0) === "/") {
+			text = "say " + text;
+		}
```

Comment:
I'd go a tiny bit differently for this.

First of all, remove the `.trim()` L271/R272 (or replace it with a [`.replace(/~+$/, '')`](https://stackoverflow.com/questions/8141718/javascript-need-to-do-a-right-trim) if you think it would be harmful to keep trailing whitespaces). That allows for another common way of escaping `/cmd`: ` /cmd`. It's frankly annoying that The Lounge currently takes the liberty of deciding that leading whitespaces should be removed.

Secondly, I'd suggest replacing this block with the following:

```js
// This is either a normal message or a command escaped with a leading `/`
if (text.charAt(0) !== "/" || text.charAt(1) === "/") {
  text = "say " + text.replace(/^\//, "");
} else {
  text = text.substr(1);
}
```

It seems more straightforward to me and the regex is basic enough to keep this readable (I'm on that "Regexes obfuscate your code team in general :-) ). What do you think?

Anyway, nice escaping the slash with another slash, and removing the trim in addition will leave an entire set of choices to the users to escape their stuff.